tests/expectations/tests/381-decltype-alias.rs

```diff
@@ -10,3 +10,6 @@ pub struct std_allocator_traits<_Alloc> {
     pub _address: u8,
     pub _phantom_0: ::std::marker::PhantomData<_Alloc>,
 }
+impl <_Alloc> Default for std_allocator_traits<_Alloc> {
```

Comment:
Probably we shouldn't derive default for incomplete/unsized types, but i understand it's needed because of derive bounds with template parameters, right?